operator/controllers/kyma_controller.go

```diff
@@ -253,7 +263,14 @@ func (r *KymaReconciler) HandleProcessingState(ctx context.Context, kyma *v1alph
 		return r.UpdateStatusWithEventFromErr(ctx, kyma, v1alpha1.StateError,
 			fmt.Errorf("error while syncing conditions during deleting non exists modules: %w", err))
 	}
-
+	statusUpdateRequiredFromSKRWebhookSync := false
+	if r.EnableKcpWatcher && kyma.Spec.Sync.Enabled {
```

Comment:
Is there a case where the watcher loop has to run even if the Kyma CR sync is disabled? E.g. for module manager notifications?

Reply:
We don't want to install the chart for kymas that have sync disabled because the watcher is not relevant for them.